.devcontainer/scripts/onCreateCommand.sh

```diff
@@ -7,5 +7,10 @@ set -e
 # restore libs tests so that the project is ready to be loaded by OmniSharp
 ./build.sh libs.tests -restore
 
+# prebuild for WASM, so it is ready for wasm development
+make -C src/mono/wasm provision-wasm
+export EMSDK_PATH=$PWD/src/mono/wasm/emsdk
+./build.sh mono+libs -os Browser -c release
```

Comment:
What happens to the library dev experience when you build `mono+libs` after building `libs+clr -rc Release`? When you `dotnet test` in a library project, does it still pick coreclr by default?

cc @safern

Reply:
I think we should be safe because the configuration will be `Browser-wasm` but let me double check.

Reply:
Theoretically, the desktop runtime should still be coreclr. But I will check to make sure too.

Reply:
Confirmed that the desktop runtime still defaults to coreclr.